src/lib.rs

```diff
@@ -654,6 +654,29 @@ impl<L, R> Either<L, R> {
             Either::Left(l) => panic!("{}: {:?}", msg, l),
         }
     }
+
+    /// Converts to a common type
```

Comment:
```suggestion
    /// Convert the contained value into `T`
```

I know we have some methods where we let the guard down and miswrote the doc, but method docs should be active, not passive, so "Convert" not "Converts". Yes I comment about useless stuff sometimes :slightly_frowning_face: